packages/cubejs-postgres-driver/driver/PostgresDriver.js

```diff
@@ -51,6 +51,24 @@ class PostgresDriver extends BaseDriver {
     }
   }
 
+  async uploadTable(table, columns, tableData) {
+    if (!tableData.rows) {
+      throw new Error(`${this.constructor} driver supports only rows upload`);
+    }
+    await this.createTable(table, columns);
+    try {
+      await this.query(
+        `INSERT INTO ${table}
+      (${columns.map(c => this.quoteIdentifier(c.name)).join(', ')})
+      SELECT * FROM UNNEST (${columns.map((c, columnIndex) => `${this.param(columnIndex)}::${c.type}[]`).join(', ')})`,
```

Comment:
@landyrev Cast in SQL is still there. I believe there's some case when `this.param(columnIndex)` doesn't work. Just curious what the case is it? If cast is required we need to use `this.fromGenericType(c.type)` because `c.type` is a generic type.

Reply:
Got it.

When you omit explicit type casting in UNNEST function you get `unnest(unknown) is not unique` error for some reason. Here are some related issues: [one](https://stackoverflow.com/questions/52678372/pq-function-unnestunknown-is-not-unique), [two](https://github.com/brianc/node-postgres/issues/1776)).

Reply:
Added `this.fromGenericType`

Reply:
@landyrev Gotcha. Yeah. Was thinking about the same. Thanks!